docs/usecases.md

```diff
@@ -8,6 +8,15 @@ sidebar_label: Use cases
 OpenEBS is used as a solution for the persistent storage needs for many stateful applications on Kubernetes. 
 
 Following are a few examples.
+- [Self Managed Database Service like RDS](#self-managed-database-service-like-rds) 
+- [Open source durable storage for Observability stack](#open-source-durable-storage-for-observability-stack)
+- [Running CI/CD on Kubernetes](#running-ci-cd-on-kubernetes)
+- [Self managed Object storage service](#self-managed-object-storage-service)
+- [Building scalable websites and ML pipelines](#building-scalable-websites-and-ml-pipelines)
+
+:::tip
+For information on how OpenEBS is used in production,  visit the [use cases](/docs/next/usecases.html) section or see examples of users and their stories on the OpenEBS Adopters section [here](https://github.com/openebs/openebs/blob/master/ADOPTERS.md) where you can also share your experience.
+:::
 
 ### Self Managed Database Service like RDS 
 
```

Comment:
Also, if a company has a large amount of data, for instance, there can be significant cost overhead to using a managed database solution available from cloud providers. -> Also, if a company has a large amount of data, for instance, there can be significant cost overhead for using a managed database solution available from cloud providers.